lib/js-api/src/api/gs-core-rest-api/api/legal-hold.api.ts

```diff
@@ -52,4 +53,64 @@ export class LegalHoldApi extends BaseApi {
             returnType: HoldPaging
         });
     }
+
+    /**
+     * Assign hold of legal holds
+     *
+     * @param holdId The identifier of a hold.
+     * @param ids one element list with id of hold to assign to existing hold
+     * @returns Promise<HoldEntry>
+     */
+    assignHold(ids: [string], holdId: string): Promise<HoldEntry> {
```

Comment:
Having two methods that are exactly the same except return type doesn't have too much sense, you can keep one of them and have return type defined like `Promise<HoldPaging | HoldEntry>` there should be some example of similar methods in the repo